src/content/structured/components/breadcrumbs/code.mdx

```diff
@@ -21,33 +21,270 @@ tabs:
 
 import { IcBreadcrumbGroup, IcBreadcrumb } from "@ukic/react";
 
+import { NavLink, MemoryRouter } from "react-router-dom";
+
 ## Component demo
 
-export const snippetsDefault = [
+export const snippets = [
   {
     language: "Web component",
     snippet: `<ic-breadcrumb-group>
-    <ic-breadcrumb page-title="Home" href="#"></ic-breadcrumb>
-    <ic-breadcrumb page-title="Beverages" href="# "></ic-breadcrumb>
-    <ic-breadcrumb current="true" page-title="Coffee" href="#"></ic-breadcrumb>
+  <ic-breadcrumb page-title="Home" href="#" />
+  <ic-breadcrumb page-title="Beverages" href="#" />
+  <ic-breadcrumb current="true" page-title="Coffee" href="#" />
 </ic-breadcrumb-group>`,
   },
   {
     language: "React",
     snippet: `<IcBreadcrumbGroup>
-    <IcBreadcrumb pageTitle="Home" href="#"></IcBreadcrumb>
-    <IcBreadcrumb pageTitle="Beverages" href="# "></IcBreadcrumb>
-    <IcBreadcrumb current="true" pageTitle="Coffee" href="#"></IcBreadcrumb>
+  <IcBreadcrumb pageTitle="Home" href="#" />
+  <IcBreadcrumb pageTitle="Beverages" href="# " />
+  <IcBreadcrumb current pageTitle="Coffee" href="#" />
 </IcBreadcrumbGroup>`,
   },
 ];
 
-<ComponentPreview snippets={snippetsDefault}>
+<ComponentPreview snippets={snippets}>
   <IcBreadcrumbGroup>
     <IcBreadcrumb pageTitle="Home" href="#" />
     <IcBreadcrumb pageTitle="Beverages" href="# " />
-    <IcBreadcrumb current="true" pageTitle="Coffee" href="#" />
+    <IcBreadcrumb current pageTitle="Coffee" href="#" />
   </IcBreadcrumbGroup>
 </ComponentPreview>
 
+## Breadcrumb details
+
+<ComponentDetails component="ic-breadcrumb" />
+
+## Breadcrumb group details
+
 <ComponentDetails component="ic-breadcrumb-group" />
+
+## Variants
+
+### Slotted links
+
+export const slottedLinks = [
+  {
+    language: "Web component",
+    snippet: `<ic-breadcrumb-group>
+  <ic-breadcrumb page-title="home">
+    <ic-link>
+      <a slot="router-item" href="#">Home</a>
+    </ic-link>
+  </ic-breadcrumb>
+  <ic-breadcrumb page-title="beverages">
+    <ic-link>
+      <a slot="router-item" href="#">Beverages</a>
+    </ic-link>
+  </ic-breadcrumb>
+  <ic-breadcrumb current="true" page-title="coffee">Coffee</ic-breadcrumb>
+</ic-breadcrumb-group>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcBreadcrumbGroup>
+  <IcBreadcrumb>
+    <IcLink>
+      <a slot="router-item" href="/">Home</a>
+    </IcLink>
+  </IcBreadcrumb>
+  <IcBreadcrumb>
+    <IcLink>
+      <a slot="router-item" href="/">Beverages</a>
+    </IcLink>
+  </IcBreadcrumb>
+  <IcBreadcrumb current>
+    <IcBreadcrumb pageTitle="Coffee" href="#" current />
+  </IcBreadcrumb>
+</IcBreadcrumbGroup>
+`,
+  },
+];
+
+<ComponentPreview snippets={slottedLinks}>
+  <IcBreadcrumbGroup>
+    <IcBreadcrumb pageTitle="Home" href="#" />
```

Comment:
There's something wrong with the slotted links. This is how the slotted links in the breadcrumbs renders
<img width="570" alt="Screenshot 2022-11-28 at 17 54 04" src="https://user-images.githubusercontent.com/113986285/204347778-de207a48-9316-4573-8ce1-a004a59d2c6d.png">

Reply:
Actually after having another look, I think it's because the 'slot="router-item"' needs to be on the IcLink rather than the `<a>` - does that work?

Reply:
I had a look at the code for [ic-link](https://github.com/mi6/ic-ui-kit/blob/main/packages/web-components/src/components/ic-link/ic-link.tsx). router-item is defo a slot within IcLink so slot="router-item" goes on the <a>. I think the issue is the slot for router item hasn't been declared using @slot

Reply:
I'll change the code back to using the slotted links and update the link file in the ic-ui-kit

Reply:
Ah ok, for some reason I was thinking "router-item" was a slot in the ic-breadcrumb component. Alright, thanks - that sounds good!

Reply:
Hey, I've removed the slotted links example because I don't think the issue is the router item slot not being declared. I'll still work on the ticket for declaring the router-item slot but this work would be around documenting the purpose of this slot. I can't figure out why it's not rendering properly as it renders fine in Storybook